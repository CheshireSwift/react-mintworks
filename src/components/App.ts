import { h } from "react-hyperscript-helpers";
import { createStore } from "redux";
import { Provider } from "react-redux";

import { rootReducer } from "../state";
import Layout from "./Layout";

const App = () => h(Provider, { store: createStore(rootReducer) }, [h(Layout)]);

export default App;
