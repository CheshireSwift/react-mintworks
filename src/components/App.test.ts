import { render, mount } from 'enzyme';
import { h } from 'react-hyperscript-helpers';

import App from './App';

describe('the app', () => {
  it('matches the snapshot', () => {
    expect(render(h(App))).toMatchSnapshot();
  });
});
