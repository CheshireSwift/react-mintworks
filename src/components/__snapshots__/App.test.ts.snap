// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the app matches the snapshot 1`] = `null`;
